.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
```

Comment:
1. It would be great if instead of `eslint:recommended`, we use the prettier rules. I believe there is a different package for that.
2. Please make sure this is consistent with the prettier rules.
3. `"parserOptions": { "ecmaVersion": 2020 }" might help based on some of our test code style.

Reply:
please explain how it is resolved

Reply:
installed eslint-plugin-prettier package and extended prettier